src/config/localization/translations.json

```diff
@@ -1552,5 +1552,21 @@
   "Removing it will suspend your profile, and you won’t be able to earn points, participate in team activities, or be eligible for new NFT drops.": "Removing it will suspend your profile, and you won’t be able to earn points, participate in team activities, or be eligible for new NFT drops.",
   "Traits": "Traits",
   "Trait": "Trait",
-  "Count": "Count"
+  "Count": "Count",
+  "$%amount%": "$%amount%",
```

Comment:
lol 🤣 
Is it needed? I think we've been putting $ before number in 9999 other places all this time

Reply:
Listen it's either get told off by @Chef-Cheems for doing this - or get told off by @memoyil for not doing it! 

Agree it's overkill. I'm sure people who put the symbol after the number can live without it.